data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -64,6 +64,30 @@
                   "type": "null"
                 }
               ]
+            },
+            "idleTimeout": {
+              "description": "The maximum time (in minutes) a connection can stay idle in the pool. A value of 0 means that idle connections are never removed from the pool.",
+              "type": "integer",
+              "minimum": 0,
+              "default": 2
+            },
+            "maxLifetime": {
+              "description": "The maximum time (in minutes) a connection can stay in the pool, regardless if it is idle or not.",
+              "type": "integer",
+              "minimum": 1,
+              "default": 30
+            },
+            "keepaliveTime": {
+              "description": "The interval time (in seconds) in which connections will be tested for aliveness, keeping them alive by the act of checking. A value of 0 means this is disabled.",
```

Comment:
instead of `keeping them alive by the act of checking` perhaps consider a new sentence along the lines of: `Connections which are no longer alive are removed from the pool`?

Reply:
Applied